### PR TITLE
PT-BR changes to the main branch

### PR DESCRIPTION
**Pull request by edsoncelio**
### Describe your changes
PT-BR changes to the main branch

### Related issue number or link (ex: `resolves #issue-number`)
N/A

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/pt-br/data-center.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Data Center
 status: Feedback Appreciated
 category: tecnologia
-tags: ["", "", ""]
+tags: ["infraestrutura", "", ""]
```
Comment (MrErlison) [inline content/pt-br/data-center.md:5]: Add "fundamento"

File: content/pt-br/debugging.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Debugging
 status: Completed
 category: conceito
-tags: ["aplicação", "", ""]
+tags: ["aplicação", "metodologia", ""]
```
Comment (MrErlison) [inline content/pt-br/debugging.md:5]: Deprecated

File: content/pt-br/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Kubernetes
+status: Completed
+category: tecnologia
+tags: ["infraestrutura", "fundamental", ""]
```
Comment (MrErlison) [inline content/pt-br/kubernetes.md:5]: "fundamento" instead of "fundamental"

File: content/pt-br/microservices.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: Microsserviços
+status: Completed
+category: conceito
+tags: ["arquitetura", "", ""]
```
Comment (MrErlison) [inline content/pt-br/microservices.md:5]: Add "fundamento"

File: content/pt-br/monolithic-apps.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: Aplicações Monolíticas
+status: Completed
+category: conceito
+tags: ["arquitetura", "", ""]
```
Comment (MrErlison) [inline content/pt-br/monolithic-apps.md:5]: Add "fundamento"

File: content/pt-br/observability.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+title: Observabilidade
+status: Completed
+category: conceito
+tags: ["metodologia", "aplicação", "infraestrutura"]
```
Comment (MrErlison) [inline content/pt-br/observability.md:5]: Now it's only "propriedade"

**Comment by MrErlison** [comment]
Hi @edsoncelio! 

Thank you very much for your contribution. 

Just a small fix.

**Comment by seokho-son** [comment]
Let me know when this PR needs to be merged. :) 
I will help the team merge this PR as a maintainer (for main branch).

**Comment by edsoncelio** [comment]
Hi @seokho-son could you help us with this PR?

**Review by MrErlison — APPROVED** [review]

**Comment by edsoncelio** [comment]
@seokho-son could you help us with this PR, please? :D

**Review by seokho-son — APPROVED** [review]
LGTM!! Thanks all! :)

**Pull request merged**